osc-server/src/test/java/org/osc/core/broker/service/servicefunctionchain/AddServiceFunctionChainServiceTest.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package org.osc.core.broker.service.servicefunctionchain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.osc.core.broker.service.dto.BaseDto;
+import org.osc.core.broker.service.request.AddOrUpdateServiceFunctionChainRequest;
+import org.osc.core.broker.service.response.BaseJobResponse;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AddServiceFunctionChainServiceTest extends BaseServiceFunctionChainServiceTest {
+
+	@InjectMocks
+	private AddServiceFunctionChainService service;
+	
+	AddOrUpdateServiceFunctionChainRequest request;
+	
+	@Override
+	@Before
+	public void testInitialize() throws Exception {
+		super.testInitialize();
+		this.service.validator = this.validatorMock;
+		request = new AddOrUpdateServiceFunctionChainRequest();
+	}
+
+	@Test
+	public void testDispatch_WithNullRequest_ThrowsNullPointerException() throws Exception {
```

Comment:
can you add tests for when validation fails?

Reply:
We already Validated the failed test case in testValidate_WhenSfcNameAlreadyExist_ThrowsVmidcBrokerValidationException with testing validation method. Did not add here thinking it will be a duplicate